components/02-molecules/banner/grand-hero/yds-grand-hero.twig

```diff
@@ -0,0 +1,58 @@
+{#
+ # Available Props:
+ # - grand_hero__content__background
+ # - grand_hero__overlay_variation: contained (default), full
+ #
+ # Available Variables:
+ # - grand_hero__heading
+ # - grand_hero__snippet
+ # - grand_hero__link__content
+ # - grand_hero__link__url
+ #
+ # Available Blocks:
+ # - grand_hero__media
+ #}
+
+{% set grand_hero__base_class = 'grand-hero-banner' %}
+{% set grand_hero__content__background = grand_hero__content__background|default('basic-black') %}
+{% set grand_hero__overlay_variation = grand_hero__overlay_variation|default('contained') %}
+
+{% set grand_hero__attributes = {
+  'data-grand-hero-overlay-variation': grand_hero__overlay_variation,
+  class: bem(grand_hero__base_class),
+} %}
+
+<div {{ add_attributes(grand_hero__attributes) }}>
+  <div {{ bem('content-wrapper', [], grand_hero__base_class) }}>
+    <div {{ bem('image', [], grand_hero__base_class) }}>
+      {% block grand_hero__image %}
+        {% include "@atoms/images/image/_responsive-image.twig" %}
+      {% endblock %}
+    </div>
+    <div {{ bem('content', [], grand_hero__base_class) }} data-component-theme={{ grand_hero__content__background }}>
+      <div {{ bem('content-inner', [], grand_hero__base_class) }}>
+        <div {{ bem('text', [], grand_hero__base_class) }}>
+          {% include "@atoms/typography/headings/yds-heading.twig" with {
+            heading__level: '1',
```

Comment:
This can't be an `h1`. It won't be the page title. I think that it can always be an `h2` though, since it will always be toward the top of the page, and won't be allowed lower in the content.

Reply:
I'm not sure about the accessibility of it though... since the actual `h1` of the page will come after this banner in the markup... My quick test shows that it'll be okay... but we'll have their accessibility team check it to be sure.

Reply:
Oh, and I just realized that you might have gotten this pattern from the action banner... which also uses an `h1`... that was originally going to _replace_ the page title, when it was used. But they decided they wanted to have the page title be shown _in addition to_ the banner. So, would you please also change the action banner heading to be an `h2` as a part of this PR?